paymentsheet/src/main/java/com/stripe/android/paymentsheet/forms/FormFieldValues.kt

```diff
@@ -5,4 +5,8 @@ import com.stripe.android.paymentsheet.specifications.IdentifierSpec
 /**
  * The identifier here comes from the form element (section, static text, etc)
  */
-class FormFieldValues(val fieldValuePairs: Map<IdentifierSpec, String?> = mapOf())
+class FormFieldValues(
+    val fieldValuePairs: Map<IdentifierSpec, String?> = mapOf(),
+    val saveForFutureUse: Boolean,
+    val showsMandate: Boolean
```

Comment:
seems this is not used anywhere, is it intended?

Reply:
It will be, @brnunes-stripe will be adding it.